hadoop-tools/hadoop-aws/src/test/java/org/apache/hadoop/fs/s3a/TestS3AExceptionTranslation.java

```diff
@@ -61,7 +63,7 @@ public class TestS3AExceptionTranslation {
   public void test301ContainsEndpoint() throws Exception {
     String bucket = "bucket.s3-us-west-2.amazonaws.com";
     int sc301 = 301;
-    AmazonS3Exception s3Exception = createS3Exception("wrong endpoint", sc301,
+    S3Exception s3Exception = createS3Exception("wrong endpoint", sc301,
```

Comment:
if the error no longer contains the endpoint, should we remove it from here too?

Reply:
Yes, I only left it until we decide on the TODO